pkgs/leak_tracker_flutter_testing/lib/src/matchers.dart

```diff
@@ -21,9 +21,9 @@ Future<List<ObjectEvent>> memoryEvents(
     }
   }
 
-  MemoryAllocations.instance.addListener(listener);
+  FlutterMemoryAllocations.instance.addListener(listener);
```

Comment:
When was `FlutterMemoryAllocations` added to package:flutter? Do we need to update the min. sdk constraint for this package (in order to specify a min. sdk that contains this typedef)?